### ISSUE TITLE
Make SimpleJSON optional

### ISSUE THREAD

**Issue by baseplate-admin** [modlinltd/django-advanced-filters#181]
**Is your feature request related to a problem? Please describe.**
Hi there, It Seems that you guys bundle `simplejson` when installing the project. Is it possible not to do this? Because its yet another dependency when i have `orjson`, `pysimdjson` in my project.

**Describe the solution you'd like**
Make `SimpleJson` optional. Preferably with `django-advanced-filters[simplejson]`

**Describe alternatives you've considered**
Forking the project and making a downstream patch

**Additional context**
Now that CPython is getting faster chances are we wont need `simplejson` to make things faster unless i am missing something here

Thanks :D

**Comment by aqeelat** [comment]
I agree here. I even think simplejson should be removed since the features we need are included in stdlib.
